Data Streams - Streams Direct (#1907)
smartcontractkit/documentation: src/features/data-streams/common/asideOnChainVerification.mdx

import { Aside } from "@components"

<Aside type="caution" title="Onchain data verification">
  In this tutorial, you decode the `FullReport` payload to extract the report data and log it to the terminal. In a

>   production environment, you should verify the data onchain to ensure its integrity and authenticity. Refer to the
>   [Onchain data verification](/data-streams/reference/streams-direct-onchain-verification) guide.
> </Aside>

Reviewer: ```suggestion
  production environment, verify the data onchain to ensure the integrity and authenticity of the report. See the [Onchain data verification](/data-streams/reference/streams-direct-onchain-verification) guide for details.
</Aside>
```
